samples/bluetooth/broadcast_config_tool/README.rst

```diff
@@ -0,0 +1,676 @@
+.. _broadcast_configuration_tool:
+
+Broadcast Configuration Tool
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Broadcast Configuration Tool application implements the :ref:`BIS gateway mode <nrf53_audio_app_overview>`.
+
+In this mode, transmitting broadcast audio happens using Broadcast Isochronous Stream (BIS) and Broadcast Isochronous Group (BIG).
+The tool uses a shell interface to be able to configure the broadcast source in many different ways.
+
+The following limitations apply to this sample application:
+
+* Audio input: Pre-encoded LC3 data from an SD-card.
+* Maximum two BIG with four BIS streams each.
```

Comment:
The first time BIG and BIS is mentioned, we should write it out in full.

Reply:
It is written out on line:12